.devcontainer/run-tests.sh

```
@@ -0,0 +1,61 @@
+  echo "Entering container..."
+  docker run -it --rm -v .:/workspaces/etl -w /workspaces/etl $ARCH /bin/bash .devcontainer/run-tests.sh $ARCH inside_container
+
+elif [ "$2" = "inside_container" ] ; then
+  echo "Cross building tests..."
+  mkdir -p build-$ARCH
+  cd build-$ARCH
+  cmake -DCMAKE_TOOLCHAIN_FILE=../.devcontainer/$ARCH/toolchain-$ARCH.cmake \
+        -DBUILD_TESTS=ON -DNO_STL=ON -DETL_CXX_STANDARD=23 \
+        -DETL_USE_TYPE_TRAITS_BUILTINS=OFF -DETL_USER_DEFINED_TYPE_TRAITS=OFF -DETL_FORCE_TEST_CPP03_IMPLEMENTATION=OFF -DETL_OPTIMISATION=-O0 -DETL_ENABLE_SANITIZER=Off -DETL_MESSAGES_ARE_NOT_VIRTUAL=OFF \
+        ../test
+  export CMAKE_BUILD_PARALLEL_LEVEL=$(nproc)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)